sn_transfers/src/cashnotes/builder.rs

```diff
@@ -200,16 +206,8 @@ impl CashNoteBuilder {
         self.signed_spends.iter().collect()
     }
 
-    /// Build the output CashNotes, verifying the transaction and SignedSpends.
-    ///
-    /// See TransactionVerifier::verify() for a description of
-    /// verifier requirements.
+    /// Build the output CashNotes
     pub fn build(self) -> Result<Vec<(CashNote, NanoTokens)>> {
-        // Verify the tx, along with signed spends.
-        // Note that we do this just once for entire tx, not once per output CashNote.
-        self.spent_tx
-            .verify_against_inputs_spent(self.signed_spends.iter())?;
-
```

Comment:
We need some kind of verification here or else there is no difference between `build` and `build_without_verifying`.